### ISSUE TITLE
'SummaryWriter' object has no attribute 'add_pr_curve'

### ISSUE THREAD

**Issue by Audrius-St** [lanpa/tensorboard-pytorch#60]
I first installed tensorboardX using

`pip install tensorboardX`

Received the error message

> AttributeError: 'SummaryWriter' object has no attribute 'add_pr_curve'

Removed tensorboardX

`pip unistall tensorboardX`

Built from source

`pip install git+https://github.com/lanpa/tensorboard-pytorch`

Received the same error message.
Checked writer.py in my installation and found that add_pr_curve is indeed missing from SummaryWriter.

Any insight regarding this issue would be appreciated.

System:

TensorFlow 1.4.0 (CPU version)
TensorFlow-TensorBoard 0.4.0rc3
PyTorch 0.3
Anaconda Python 3.6
Linux: Ubuntu 17.10 with kernel 4.13.0

**Comment by lanpa** [comment]
Maybe something is not removed. Can you `pip uninstall tensorboardX` and post the output of `conda list|grep tensor` and the output of `which python`

**Comment by Audrius-St** [comment]
Thank you for your prompt response.

I had to remove and reinstall Anaconda Python 3.6 due to another unrelated corruption issue.

This time first built tensorboardX from source.

`pip install git+https://github.com/lanpa/tensorboard-pytorch`

Checked writer.py in my installation and found that add_pr_curve is now present in SummaryWriter.

Running my PyTorch model, the line with writer.ad_pr_curve was now executed.

Without identifying the exact source of the problem, your intuition that something was not removed is correct.

Thank you for your assistance.
